components/kubernetes/versions.go

```diff
@@ -0,0 +1,56 @@
+package kubernetes
+
+type kindConfig struct {
+	kindVersion string
+	nodeImage   string
+}
+
+// Source: https://github.com/kubernetes-sigs/kind/releases
+var kubeToKindVersion = map[string]kindConfig{
+	"1.27": {
+		kindVersion: "v0.20.0",
+		nodeImage:   "kindest/node:v1.27.3",
```

Comment:
I am not sure you need to repeat `kindest/node` everywhere, especially if we want to change registry.